### PR TITLE
Js fixes and updates

### PR DESCRIPTION
**Pull request by clavay**
set first calendar day to monday

check if the span field with class .boolean-value exist before writting
the value
for svg process flow diagram item the method is different so the span
does not exist
todo : replace value by string for process diagram items

check if '#aggregation-type-all-select-' + widget_id exist before adding
new option

### CODE REVIEW

File: pyscada/hmi/static/pyscada/js/pyscada/pyscada_v0-7-0rc14.js [hunk 1]
```diff
@@ -2802,7 +2802,7 @@ function createOffset(date) {
                      "November",
                      "December"
                  ],
-                 "firstDay": 1
+                 "firstDay": 0
```
Comment (trombastic) [inline pyscada/hmi/static/pyscada/js/pyscada/pyscada_v0-7-0rc14.js:2805]: this is only the default in most of europe, maybe it ist better to get that info from the locaton settings of the browser?

Comment (clavay) [reply pyscada/hmi/static/pyscada/js/pyscada/pyscada_v0-7-0rc14.js:2805]: Do you know how and compatible with many browsers ?

Comment (clavay) [reply pyscada/hmi/static/pyscada/js/pyscada/pyscada_v0-7-0rc14.js:2805]: See : https://stackoverflow.com/questions/53382465/how-can-i-determine-if-week-starts-on-monday-or-sunday-based-on-locale-in-pure-j

Comment (clavay) [reply pyscada/hmi/static/pyscada/js/pyscada/pyscada_v0-7-0rc14.js:2805]: As the current config is `"daysOfWeek": [
                     "Mo",
                     "Tu",
                     "We",
                     "Th",
                     "Fr",
                     "Sa",
                     "Su",
                 ],`
The current setting is showing `Tuesday` as the first day of the week.

So we can change it to Monday or Sunday till we found how to set it accordingly to the browser settings.

This PR propose Monday.

Comment (clavay) [reply pyscada/hmi/static/pyscada/js/pyscada/pyscada_v0-7-0rc14.js]: I add a new commit to correct a bug about naming the days.
The day list in the JS datetimepicker config should start on sunday.

I think you can approve this request to fix this bug and then we plan in github `discussion` or `project` to get the browser locale to set the first week day.

What do you think @trombastic ?

**Review by trombastic — APPROVED** [review]

File: pyscada/hmi/static/pyscada/js/pyscada/pyscada_v0-7-0rc14.js (hunk 1 above)
Comment (trombastic) [reply pyscada/hmi/static/pyscada/js/pyscada/pyscada_v0-7-0rc14.js]: Okay

**Pull request merged**